api/client.yaml

```diff
@@ -1203,6 +1203,44 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/moov-io/api/master/openapi-common.yaml#/components/schemas/Error'
+    delete:
+      description: Remove a customer's document
+      operationId: deleteCustomerDocument
+      parameters:
+        - name: X-Request-ID
```

Comment:
It seems `X-Namespace` header is missed.

Reply:
I don't think we're using `X-Namespace` for documents yet, Adam put a `TODO` for that here https://github.com/moov-io/customers/pull/184/files/ac983b68d92dcb2f2e4704465c424d9565d0dfcd#diff-991f4c656606164965e3c2d72e34946eL86

Reply:
I'm just seeing it in api/client.yaml for all enpoints.

Reply:
Whoops. I should have resolved that before my merge. We can include `X-Namespace` on the request and `inner join` with `customers` to verify it matches.

Reply:
Let's do that in another PR. Since the route currently doesn't care I'm fine merging this PR.

Reply:
Sounds good; is this PR ok to merge?